content/blog/2023/09/06/2023-09-06-artifactory-bandwidth-reduction.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Artifactory Bandwidth Reduction"
```

Comment:
```suggestion
title: "Artifactory Bandwidth Reduction: Stop Caching Apache Central"
```

Reply:
apache central is not a thing, it is Maven Central